package.json

```diff
@@ -63,6 +63,7 @@
     "grunt-exec": "0.4.6",
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
+    "grunt-cli": "0.1.13",
```

Comment:
We typically install the `grunt-cli` globally: AFAIK, this is the community-standard-practice. Let me know if there's a reason that you can't do that.